### PR TITLE
Simplify Header and App styling code

### PR DESCRIPTION
**Pull request by EjPlatzer**
The html and CSS to style the Header and the overall app are more complicated than is necessary to achieve the desired style. I have simplified this a little bit.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,139 +1,105 @@
...
+  <meta name="description" content="Gordon 360 is the student app for Gordon College." />
+</head>
 
-  <body>
-    <noscript> You need to enable JavaScript to use Gordon 360. </noscript>
-    <div id="root"><body style="background-image: linear-gradient(to right, #014983 0%, 
+<body>
+  <noscript> You need to enable JavaScript to use Gordon 360. </noscript>
+  <div id="root">
+    <div style="width: 100%; height: 100%; background-image: linear-gradient(to right, #014983 0%,
```
Comment (EjPlatzer) [inline index.html:102]: Most "changes" to this file were just autoformatting. The only contentful changes I made were to the element inside the root div. Previously, this was a duplicate `body` tag (that was unclosed) which is invalid html. I made it a `div` instead and added `width: 100%; height: 100%;` to it's inline styles.

File: src/components/Header/Header.module.scss
Comment (Centerville1) [inline src/components/Header/Header.module.scss:1]: This looks super good, and so much simpler than it was previously!  One thing I notice, on smaller screens there's a size where the tab text gets cut off slightly, plus there's also no room to add the recim tab to the header (idk what the plan is with that, but we had taken that into consideration for the tab spacing in the previous iteration)

Comment (Centerville1) [reply src/components/Header/Header.module.scss:1]: Also, the way on small screens that the people searchbar has no spacing to the header logo looks a little strange, but definitely fixes the previous sizing issues we were having.

**Review by Centerville1 — COMMENTED** [review]
Looks great, super helpful cleanup and will make this much easier to understand and work with for the future.  I noted a couple of minor spacing issues on certain screens, but otherwise lgtm!

**Comment by Centerville1** [comment]
@EjPlatzer we accidently pushed to this branch and tried to revert the changes in a seperate branch when we accidentally brought your commits into that other branch.  We'll probably have to revert those reverted commits to get these changes in, sorry for that mess!

**Comment by Centerville1** [comment]
> @EjPlatzer we accidently pushed to this branch and tried to revert the changes in a seperate branch when we accidentally brought your commits into that other branch. We'll probably have to revert those reverted commits to get these changes in, sorry for that mess!

lmk if you want me to help get those reverts made

**Pull request merged**